TezosKit/Conseil/ConseilClient.swift

```diff
@@ -170,6 +180,24 @@ public class ConseilClient {
     networkClient.send(rpc, completion: completion)
   }
 
+	/// An RPC which fetches received transactions for an account, that came from smart contracts. For example if an account reiceved some FA1.2 tokens
```

Comment:
nit: misspelling of 'received'

Reply:
I'll have you know, typos are my superpower